apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V37__Create_Schema_Idiomatic_Names.sql

```diff
@@ -0,0 +1,1787 @@
+);
+
+create index if not exists beneficiaries_hicn_idx
+	on beneficiaries (bene_crnt_hic_num);
+
+create table if not exists beneficiaries_history (
+    bene_history_id                          bigint not null,                          -- beneficiaryHistoryId
+    bene_id                                  bigint not null,                          -- beneficiaryId
+    bene_birth_dt                            date not null,                            -- birthDate
+    bene_sex_ident_cd                        character(1) not null,                    -- sex
+    bene_crnt_hic_num                        character varying(64) not null,           -- hicn
+    mbi_num                                  character varying(11),                    -- medicareBeneficiaryId
+    hicn_unhashed                            character varying(11),                    -- hicnUnhashed
+    mbi_hash                                 character varying(64),                    -- mbiHash
+    efctv_bgn_dt                             date,                                     -- mbiEffectiveDate
+    last_updated                             timestamp with time zone,                 -- lastupdated
+    constraint beneficiaries_history_pkey
+        primary key (bene_history_id),
+ 
+    constraint beneficiaries_history_bene_id_to_beneficiaries
+        foreign key (bene_id)
+            references beneficiaries (bene_id) match simple
+            on update no action
+            on delete no action
+);
+
+create index if not exists beneficiaries_history_bene_id_idx
+	on beneficiaries_history (bene_id);
+
+create index if not exists beneficiaries_history_hicn_idx
+	on beneficiaries_history (bene_crnt_hic_num);
+
+create index if not exists beneficiaries_history_mbi_hash_idx
+	on beneficiaries_history (mbi_hash);
+
+create table if not exists beneficiaries_history_invalid_beneficiaries (
+    bene_history_id                          bigint not null,                          -- beneficiaryHistoryId
+    bene_id                                  bigint,                                   -- beneficiaryId
+    bene_birth_dt                            date not null,                            -- birthDate
+    bene_sex_ident_cd                        character(1) not null,                    -- sex
+    bene_crnt_hic_num                        character varying(64) not null,           -- hicn
+    mbi_num                                  character varying(11),                    -- medicareBeneficiaryId
+    hicn_unhashed                            character varying(11),                    -- hicnUnhashed
+    constraint beneficiaries_history_invalid_beneficiaries_pkey
+        primary key (bene_history_id)
+);
+
+create table if not exists beneficiary_monthly (
+    year_month                               date not null,                            -- yearMonth
+    parent_beneficiary                       bigint not null,                          -- parentBeneficiary
```

Comment:
any thoughts of changing this to bene_id?

Reply:
thoughts by me...for sure! BUT, for wahtever reason, we have db columns using that 'parent_' naming convention (i.e., parent_claim). My pref would be to scrap that use of 'parent_' and use table-qualified names as needed; for example...
`
select d.* from carrier_claim_lines d
inner join carrier_claims c on c.clm_id = d.clm_id
inner join beneficiaries b where b.bene_id = c.bene_id
`